File: CiscoPCCE.Toolkit/Bean/DataCenterInventoryFile.cs
using System.Xml.Serialization;

using CiscoPCCE.Toolkit.Sdk;

namespace CiscoPCCE.Toolkit.Bean
{
    [XmlRoot("datacenter")]
    public class DataCenterInventoryFile : BaseApiBean
    {
        [XmlElement("inventoryFile")]
        public required InventoryFile InventoryFile { get; set; }

        [XmlElement("name")]
        public string? Name { get; set; }
    }
}

File: JavaSrc/toolkit/bean/DataCenterInventoryFile.java
// ----------------------------------------------
package com.cisco.ccbu.cce.unifiedconfig.toolkit.bean;

import com.cisco.ccbu.cce.unifiedconfig.toolkit.BaseApiBean;
import javax.xml.bind.annotation.XmlSeeAlso;
import javax.ws.rs.Path;
import javax.xml.bind.annotation.XmlRootElement;
import java.util.List;
import com.cisco.ccbu.cce.unifiedconfig.toolkit.ReferenceBean;
import java.util.Date;
import com.cisco.ccbu.cce.unifiedconfig.toolkit.BaseApiListBean;

import javax.xml.bind.annotation.XmlElement;

import javax.xml.bind.annotation.XmlElementWrapper;


@XmlRootElement(name="datacenter")
public class DataCenterInventoryFile extends BaseApiBean {
  private InventoryFile inventoryFile;
  private String name;

  public InventoryFile getInventoryFile() {
     return this.inventoryFile;
  }

  public void setInventoryFile(InventoryFile inventoryFile) {
     this.inventoryFile = inventoryFile;
  }

  public String getName() {
     return this.name;
  }

  public void setName(String name) {
     this.name = name;
  }


}
